test/post.test.js

```diff
@@ -23,3 +23,15 @@ t.test('cannot set schemaCompiler after binding', t => {
     }
   })
 })
+
+t.test('get schemaCompiler after set schemaCompiler', t => {
+  t.plan(2)
+  const mySchemaCompiler = () => { }
+
+  const fastify = Fastify()
+  fastify.setSchemaCompiler(mySchemaCompiler)
+  const sc = fastify.getSchemaCompiler()
```

Comment:
I think we should refactor these to be getter and setters for the schemaCompiler property.

Reply:
Do you mean to add the setter removing the `setSchemaCompiler`?
Wouldn't this be a breaking change?